apps/nextra/pages/en/network/blockchain/execution.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Parallel Execution"
+---
+
+# Execution
+
+In blockchains, execution refers to the process by which validators 
+run and apply smart contract transactions from ordered blocks to update the blockchain’s state.
+
+It is crucial because validators must agree on the same final state, post-execution,
+ensuring consistency and security across the network.
+
+Efficient execution, particularly parallel execution, is key to scaling blockchain performance and throughput.
+
+## Why Parallel Execution?
+
+![](https://miro.medium.com/v2/resize:fit:4800/format:webp/0*qG0KRe1KzsP8Odth)
+
+Smart contract execution is a major throughput bottleneck for blockchains. 
+After proposing blocks and agreeing on their order, validators have to execute the 
+transactions in the ordered blocks. Crucially, validators must arrive at the same final state, 
+which must correspond to some sequential execution of the transactions. 
+Due to the lack of a foundational solution, current blockchains either execute 
+sequentially or require an embarrassingly parallel workload (i.e. no inner conflicts) 
+for improved performance. Sequential execution does not scale well, and assuming that 
+all transactions are commutative is unrealistic when considering a broad range of smart contracts. 
```

Comment:
This was mostly a complex rewording of the earlier statements.  Validators, ordering, etc. are very specific to specific blockchains, how about this for a reword, focusing on the story behind, sequential, pessimistic parallelism
```suggestion
Execution is a major throughput bottleneck for blockchains.  In a [BFT](https://aptos.dev/en/network/glossary#byzantine-fault-tolerance-bft) blockchain, validators propose blocks, and execute the transactions within in a specific order.  It is important that the final state is the same across each validator and the final state is the same as the order.

The simplest approach is to execute all transactions in order (sequential).  However, this does not scale well.  Throughput drops because only one transaction can be executed at a time and have varying execution lengths.

The next approach is to require developers to specify the conflicts between transactions.  The VM would then need to execute transactions in parallel for those that do not conflict, also known as pessimistic parallelism.  This brings a higher burden on the developer, and in many cases force transactions that don't actually conflict, to be sequential.  Having a large number of sequential transactions, especially with varying execution times, can cause the latency and throughput of the blockchain to drop drastically.
```

Reply:
We call this static parallelism.